app/clients/sms/mmg.py

```diff
@@ -77,22 +77,6 @@ def init_app(self, current_app, statsd_client, *args, **kwargs):
         self.statsd_client = statsd_client
         self.mmg_url = current_app.config.get('MMG_URL')
 
-    def record_outcome(self, success, response):
-        status_code = response.status_code if response else 503
-        log_message = "API {} request {} on {} response status_code {}".format(
```

Comment:
we currently rely on finding these logs lines in cloudwatch to gather historical stats. i'm kinda interested in moving away from this method anyway since the cloudwatch insight always times out so is an approximation anyway, but we'd need something in place before we can change this log line.

https://github.com/alphagov/notifications-manuals/wiki/Support-Runbook#provider-5xx-vs-2xx-stats

Reply:
Great catch 💯.

As discussed in Slack, we can still make future reports work by parsing logs based on "success" and "failed". We're about to start a new quarter, so now is the ideal time to change the reporting.

I asked Caley about this and it's unlikely we'll need reports for previous quarters. I can save the old query and mention it in the runbook - I assume we can just ditch it after a year or so.

Does that sound OK?

Reply:
Update: we agreed it's OK to change the logging at the start of the quarter so long as we update the query and runbook to match, and have a way to query the old data if we need to.